BitwardenShared/UI/Auth/CompleteRegistration/MasterPasswordGenerator/MasterPasswordGeneratorView.swift

```diff
@@ -0,0 +1,103 @@
+import SwiftUI
+
+// MARK: - MasterPasswordGeneratorView
+
+/// A view that allows the user to generate a master password.
+///
+struct MasterPasswordGeneratorView: View {
+    // MARK: Properties
+
+    /// The `Store` for this view.
+    @ObservedObject var store: Store<
+        MasterPasswordGeneratorState,
+        MasterPasswordGeneratorAction,
+        MasterPasswordGeneratorEffect
+    >
+
+    // MARK: View
+
+    var body: some View {
+        VStack(spacing: 24) {
+            passwordTextField
+
+            generateButton
+
+            instructionsPreventAccountLockView
+        }
+        .scrollView()
+        .navigationBar(
+            title: Localizations.generateMasterPassword,
+            titleDisplayMode: .inline
+        )
+        .toolbar {
+            cancelToolbarItem {
+                store.send(.dismiss)
+            }
+            saveToolbarItem {
+                await store.perform(.save)
+            }
+        }
+        .task {
+            await store.perform(.loadData)
+        }
+    }
+
+    // MARK: Private views
+
+    /// The generated password text field.
+    private var passwordTextField: some View {
+        BitwardenTextField(text: store.binding(
+            get: \.generatedPassword,
+            send: MasterPasswordGeneratorAction.masterPasswordChanged
+        ))
+        .submitLabel(.done)
```

Comment:
🤔 Should this be an actual text field that you can edit? I guess I assumed it would be read-only and use `PasswordText`.

Reply:
yeah good catch!  figma shows it with the `PasswordText` look as well.  Making the update